File: 백준/Bronze/31775. 글로벌 포닉스/글로벌 포닉스.cs
using System;

class Program
{
    static void Main()
    {
        string[] Global_Ponix = new string[3];
        for (int i = 0; i < 3; i++)
        {
            Global_Ponix[i] = Console.ReadLine();
        }

        bool l = false, k = false, p = false;

        foreach (var i in Global_Ponix)
        {
            if (i[0] == 'l')
                l = true;
            else if (i[0] == 'k')
                k = true;
            else if (i[0] == 'p')
                p = true;
        }

        if (l && k && p)
            Console.WriteLine("GLOBAL");
        else
            Console.WriteLine("PONIX");
    }
}

File: 백준/Bronze/31775. 글로벌 포닉스/글로벌 포닉스.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        String[] Global_Ponix = new String[3];
        for (int i = 0; i < 3; i++) {
            Global_Ponix[i] = sc.nextLine();
        }

        boolean l = false, k = false, p = false;

        for (String i : Global_Ponix) {
            if (i.charAt(0) == 'l')
                l = true;
            else if (i.charAt(0) == 'k')
                k = true;
            else if (i.charAt(0) == 'p')
                p = true;
        }

        if (l && k && p)
            System.out.println("GLOBAL");
        else
            System.out.println("PONIX");
    }
}
